docs/layouts/shortcodes/generated/expert_forst_section.html

```diff
@@ -0,0 +1,54 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>state.backend.forst.executor.inline-coordinator</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Whether to let the task thread be the coordinator thread responsible for distributing requests.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.executor.inline-write</h5></td>
+            <td style="word-wrap: break-word;">true</td>
+            <td>Boolean</td>
+            <td>Whether to let write requests executed within the coordinator thread.</td>
```

Comment:
It would be good to have some guidance as to when this should be set or not.

Reply:
Added.